assets/js/components/setup/compatibility-checks.js

```diff
@@ -42,6 +42,7 @@ const ERROR_FETCH_FAIL = 'check_fetch_failed';
 const ERROR_TOKEN_MISMATCH = 'setup_token_mismatch';
 const ERROR_GOOGLE_API_CONNECTION_FAIL = 'google_api_connection_fail';
 const ERROR_AMP_CDN_RESTRICTED = 'amp_cdn_restricted';
+const ERROR_OLD_VERSION = 'old_wordpress';
```

Comment:
The name here isn't that important but I think we can be a bit more clear with the intent with a more self-documenting name here since "old" is a bit vague.

How about this?
```suggestion
const ERROR_WP_PRE_V5 = 'wp_pre_v5';
```